src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--30">
+      <ul class="calendar__month">
```

Comment:
Do you need extra wrapper here? Let's get rid of it, you've already have `div` as wrapper for the calendar